src/server/routes/changePassword.ts

```diff
@@ -68,6 +69,9 @@ router.get(
   `${Routes.CHANGE_PASSWORD}${Routes.CHANGE_PASSWORD_TOKEN}`,
   async (req: Request, res: ResponseWithRequestState) => {
     const { token } = req.params;
+    res.locals.browserName = getBrowserNameFromUserAgent(
```

Comment:
to the previous comment, could this become `res.locals.pageData.browserName = ..`?

i suppose this depends on the outcome of the work that @paddyo is doing around immutability